docs/.kratos.yaml

```diff
@@ -68,12 +68,14 @@ selfservice:
     after:
       password:
         - run: session
+        - run: revoke_active_sesions
```

Comment:
Typo, this should be `revoke_active_sessions`

Reply:
done